deeppavlov/core/models/keras_model.py

```diff
@@ -170,4 +173,66 @@ def __call__(self, *x_batch, **kwargs) -> Union[List, np.ndarray]:
         """
         with self.graph.as_default():
             K.set_session(self.sess)
-            return self._net.predict_on_batch(x_batch, **kwargs)
\ No newline at end of file
+            return self._net.predict_on_batch(x_batch, **kwargs)
+
+
+class LRScheduledKerasModel(LRScheduledModel, KerasModel):
+    """
+    KerasModel enhanced with optimizer, learning rate and momentum
+    management and search.
+    """
+    def __init__(self, **kwargs):
+        if isinstance(kwargs.get("learning_rate_decay"), str):
+            LRScheduledModel.__init__(self, **kwargs)
```

Comment:
Why only `str` typed `learning_rate_decay` is supported?

Reply:
yes, you are right. decay for schedule also could be a list.
If learning rate decay is float and learning rate is float than will pass to keras_model, otherwise, lr scheduled model